src/conversion/conversion.js

```diff
@@ -402,17 +405,16 @@ export default class Conversion {
 	 */
 	attributeToElement( definition ) {
 		// Set up downcast converter.
-		this.for( 'downcast' ).add( downcastAttributeToElement( definition ) );
+		this.for( 'downcast' ).attributeToElement( definition );
 
 		// Set up upcast converter.
 		for ( const { model, view } of _getAllUpcastDefinitions( definition ) ) {
-			this.for( 'upcast' ).add(
-				upcastElementToAttribute( {
+			this.for( 'upcast' )
+				.elementToAttribute( {
 					view,
 					model,
 					converterPriority: definition.priority
```

Comment:
Since methods have the same names now, shouldn't they have the same parameters names too?

Reply:
I mean `converterPriority` vs `priority`, in `conversion.for( 'downcast' ).attributeToElement` vs `conversion.attributeToElement`.